### PR TITLE
chore: Tidy up our soak runs

### PR DESCRIPTION
**Pull request by blt**
This commit tidies up the capture files etc from soak runs, resolving #12184.

Signed-off-by: Brian L. Troutwine <brian@troutwine.us>

### CODE REVIEW

**Comment by blt** [comment]
I've included unrelated changes here to experiment with. Notably, I'm using GH hosted runners to avoid congesting the main runner pool.

**Comment by blt** [comment]
> I've included unrelated changes here to experiment with. Notably, I'm using GH hosted runners to avoid congesting the main runner pool.

Now removed.

**Review by JeanMertz — APPROVED** [review]

File: .github/workflows/soak.yml [hunk 1]
```diff
@@ -270,9 +271,9 @@ jobs:
           name: ${{ github.run_id }}-${{ github.run_attempt }}-${{ matrix.target }}-baseline
           path: /tmp/${{ github.run_id }}-${{ github.run_attempt }}/${{ matrix.target }}/
 
-      - name: Clear up unused images
+      - name: Clean up machine
         run: |
-          minikube delete --all --purge
+          rm -rf /tmp/${{ github.run_id }}-${{ github.run_attempt }}/${{ matrix.target }}/
```
[the remark this answers is not in the document]
Comment (jszwedko) [reply .github/workflows/soak.yml:276]: I think this was vestigial since the soak tests no longer use minikube.

**Review by jszwedko — APPROVED** [review]
Thanks!

File: .github/workflows/soak.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (blt) [reply .github/workflows/soak.yml:276]: Good eye. Yes, minikube was removed in https://github.com/vectordotdev/vector/pull/11891 and this was vestigial.

**Pull request merged**